services/Postmates.mjs

```diff
@@ -181,5 +181,59 @@ class Postmates extends Service {
       throw new HTTPResponseError(res);
     }
   }
+  /* Get restaraunt menu
+   * @param {String} contains the restraunt ID (Should come from search results as storeUUID)
+   * @return {Array} an array of [storeName, storeID, storeImage, storeHours,
+   * menu[category[items[name, description, price, image]]]], or HTTPResponseError
+   */
+
+  async getMenu(restarauntID) {
+    const res = await fetch("https://postmates.com/api/getStoreV1", {
+      method: "POST",
+      headers: {
+        authority: "postmates.com",
+        accept: "*/*",
+        "content-type": "application/json",
+        dnt: "1",
+        "x-csrf-token": "x",
+      },
+      body: '{"storeUuid":"' + restarauntID + '"}',
```

Comment:
Lets keep the body's the same for these external API calls to make them easier to debug in the future by using JSON.stringify like follows:

```javascript
JSON.stringify({storeUuid:  restarauntID})
```

Reply:
Good suggestion, fixed